homeassistant/components/homewizard_energy/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Homewizard Energy",
+  "config": {
+    "step": {
+      "confirm": {
+        "description": "[%key:common::config_flow::description::confirm_setup%]"
+      }
+    },
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]"
```

Comment:
these are not used. already_configured is used.

Reply:
Added used translations and removed unused translations with https://github.com/DCSBL/core/commit/13b652b581b45f4cf15cd4c06704f2af0f2b8359